installer/config/Conf.tpl.php

```diff
@@ -28,12 +28,15 @@ class Conf
     public function __construct()
     {
         $this->dbHost = '{{dbHost}}';
+        //$this->dbHost = php_sapi_name() === 'cli' ? '127.0.0.1' : 'mariadb'; // Uncomment if the installation doesn't complete
+
         $this->dbPort = '{{dbPort}}';
         if (defined('ENVIRONMENT') && ENVIRONMENT == 'test') {
             $prefix = defined('TEST_DB_PREFIX') ? TEST_DB_PREFIX : '';
             $this->dbName = $prefix . 'test_{{dbName}}';
         } else {
-            $this->dbName = '{{dbName}}';
+            $this->dbName = 'orangehrm'; 
+            // $this->dbName = '{{dbName}}'
         }
```

Comment:
I dislike hardcoding prod dbName here, maybe do the next suggestion for production environment

```suggestion
        } else if (defined('ENVIRONMENT') && ENVIRONMENT == 'production') {
            $this->dbName = '{{dbName}}';
        } else {
            $this->dbName = 'orangehrm';
        }
```